source/Grid/Grid.test.js

```diff
@@ -120,43 +120,49 @@ describe('Grid', () => {
   describe('shows and hides scrollbars based on rendered content', () => {
     const scrollbarSize = getScrollbarSize()
 
+    function renderTwice (markup) {
```

Comment:
I have a little time today while flying to just pick this up. I'll base it on what you've done and make the changes I've suggested. I'll give you credit for the idea/suggestion in release notes. :)